### PR TITLE
editorial: Expand SLSA acronym in docs

### PR DESCRIPTION
**Pull request by swickersh**
When visiting slsa.dev/spec/$version/about, I didn't see any mention of what the SLSA acronym actually stands for.
This change simply expands the acronym in the docs.

### CODE REVIEW

File: docs/spec/v1.1/about.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ to SLSA, start here!
 
 ## What is SLSA?
 
+SLSA ("salsa") is Supply-chain Levels for Software Artifacts.
+
 SLSA is a set of incrementally adoptable guidelines for supply chain security,
```
Comment (joshuagl) [inline docs/spec/v1.1/about.md:13]: How do you feel about the following edit to align with the text in the "What is SLSA?" section of our landing page https://slsa.dev/ ?

```suggestion
Supply-chain Levels for Software Artifacts, or SLSA ("salsa"), is a set of incrementally adoptable guidelines for supply chain security,
```

**Review by joshuagl — COMMENTED** [review]
Thank you for taking the time to submit a PR! I've proposed slightly different phrasing, let me know what you think.

File: docs/spec/v1.1/about.md (hunk 1 above)
Comment (swickersh) [reply docs/spec/v1.1/about.md]: Even better. Updated. Thanks!

**Review by joshuagl — APPROVED** [review]
Thank you!

**Review by lehors — APPROVED** [review]
Thanks.

**Pull request merged**